support/docker/production/docker-compose.yml

```diff
@@ -51,6 +51,11 @@ services:
     networks:
       default:
         ipv4_address: 172.18.0.42
+    
+    environment: 
```

Comment:
We don't need this, inject them in `.env` file instead

Reply:
we have removed environment from docker-compose.yml file.